paper-server/src/main/java/org/bukkit/craftbukkit/attribute/CraftAttribute.java

```diff
@@ -16,6 +16,9 @@ public class CraftAttribute extends OldEnumHolderable<Attribute, net.minecraft.w
 
     private static int count = 0;
 
+    private final Holder<net.minecraft.world.entity.ai.attributes.Attribute> holder;
```

Comment:
this is necesary? you can call this.getHolder for get the already holder passed to `HolderableBase`